apps/playground/src/app/quiz/components/question/question.component.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, OnInit, OnChanges, SimpleChanges, Input, Output, EventEmitter } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+
+import { QuizQuestion } from '../../model/QuizQuestion';
+
+@Component({
+  selector: 'codelab-quiz-question',
+  templateUrl: './question.component.html',
+  styleUrls: ['./question.component.scss']
+})
+export class QuestionComponent implements OnInit, OnChanges {
+  @Output() answer = new EventEmitter<string>();
+  @Output() formGroup: FormGroup;
+  @Input() question: QuizQuestion;
+  option = '';
+  grayBorder = '2px solid #979797';

```

Comment:
please put it in the template

Reply:
added it to dependency-injection-quiz template and added scss